Content.Server/Chemistry/ReagentEffects/MakeSentient.cs

```diff
@@ -16,25 +16,16 @@ public override void Effect(ReagentEffectArgs args)
         var entityManager = args.EntityManager;
         var uid = args.SolutionEntity;
 
-        // This makes it so it doesn't affect things that are already sentient
-        if (entityManager.HasComponent<MindContainerComponent>(uid))
-        {
-            return;
-        }
-
-        // This piece of code makes things able to speak "normally". One thing of note is that monkeys have a unique accent and won't be affected by this.
+        // Let affected entities speak normally to make this effect different from, say, the "random sentience" event
+        // This also works on entities that already have a mind
         entityManager.RemoveComponent<ReplacementAccentComponent>(uid);
-
-        // Monke talk. This makes cognizine a cure to AMIV's long term damage funnily enough, do with this information what you will.
         entityManager.RemoveComponent<MonkeyAccentComponent>(uid);
 
-        // This makes it so it doesn't add a ghost role to things that are already sentient
```

Comment:
Why are you removing this comment?

Reply:
It's to follow the rule of "comment why, not what". The code is self-explanatory (as is all well-written code), so it doesn't need a what explanation, and the answer to "why is this code here" is also pretty obvious, so the comment is unneeded.

Reply:
But the existing comment *is* about *why*, all it's missing is a follow on statement of "... because that would let a player who drinks cognizine eligible for a ghost takeover." Somebody who is not familiar with mind code may not realize this possibility. Taking away this comment would remove valuable context.

Reply:
Yeah, good point, I'll add the comment back in a bit.